presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -49,20 +52,22 @@ public void runCommand()
             case RPM:
                 File tempFile;
                 try {
+                    LOGGER.debug("Downloading package from url : {}", urlToFetchPackage);
```

Comment:
For any debug message, one would be sufficient to tell the user its current command. If it succeed, then there is no need to say it out. Only error needs to be recorded.